### ISSUE TITLE
Fix drivebase code

### ISSUE THREAD

**Issue by cotc167** [icrobotics-team167/2024_Crescendo#10]

**Comment by TheComputer314** [comment]
The absolute encoder seems to be running on rotations when the PIDs expect degrees, causing the spins. Tried debugging this, but after a while ShuffleBoard completely stopped connecting to the robot, spamming
```
NT: CONNECTED NT4 client 'shuffleboard@1' (from 10.1.67.151:50298)
NT: DISCONNECTED NT4 client 'shuffleboard@1' (from 10.1.67.151:50298): incomplete fragment
```
53bf9899486aac145369c1dbe74ebc2535bef728

**Comment by TheComputer314** [comment]
Fixed it e12c4771963dd74fe520983a7f99a91536191ff7

Also more aggressive PIDs for snappier module turning 85b301646178ffc43b25c5f26dcef9ec82979da3
